### PR TITLE
fix nft background

### PR DESCRIPTION
**Pull request by skylarbarrera**
before: http://cloud.skylarbarrera.com/Screen-Shot-2021-05-26-00-05-33.86.png
after: http://cloud.skylarbarrera.com/Screen-Shot-2021-05-26-00-05-58.41.png

fixes RNBW-1143

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "Rainbow",
-  "version": "1.5.9-1",
+  "version": "1.5.10-1",
```
Comment (osdnk) [inline package.json:3]: This should not be here!

**Review by osdnk — APPROVED** [review]
Please, rebase your changes!

File: package.json (hunk 1 above)
Comment (skylarbarrera) [reply package.json:3]: oop thought i fixed this

Comment (skylarbarrera) [reply package.json]: good 2 go

**Pull request merged**